src/pages/office.tsx

```diff
@@ -41,79 +73,172 @@ const OfficePage = ({ location }: OfficePageProps) => {
+            <br />
+            Cras justo odio, dapibus ac facilisis in, egestas eget quam.
+            Praesent commodo cursus magna, vel scelerisque nisl consectetur et.
+            Morbi leo risus, porta ac consectetur ac, vestibulum at eros. Donec
+            sed odio dui. Integer posuere erat a ante venenatis dapibus posuere
+            velit aliquet. Aenean eu leo quam. Pellentesque ornare sem lacinia
+            quam venenatis vestibulum.
+          </StyledSectionHeader>
+          <ImageContentWrapper>
+            <Image
+              description="Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Morbi leo risus, porta ac consectetur ac, vestibulum at eros. "
+              textAlign="right"
+            >
+              <StaticImage
+                src="./../images/office/sy-office-03.jpg"
+                alt="kitchen"
+                aspectRatio={564 / 295}
+                width={564}
+              />
+            </Image>
+            <Image
+              description="Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Morbi leo risus, porta ac consectetur ac, vestibulum at eros. "
+              textAlign="left"
+            >
+              <StaticImage
+                src="./../images/office/sy-office-02.jpg"
+                alt="kitchen"
+                aspectRatio={564 / 295}
+                width={564}
+              />
+            </Image>
+            <Image
+              description="Integer posuere erat a ante venenatis dapibus posuere velit aliquet. Morbi leo risus, porta ac consectetur ac, vestibulum at eros. "
+              textAlign="right"
+            >
+              <StaticImage
+                src="./../images/office/sy-office-04.jpg"
+                alt="kitchen"
+                aspectRatio={564 / 295}
+                width={564}
+              />
+            </Image>
+          </ImageContentWrapper>
+          <StyledSectionHeader headline="Das sind wir" kicker="Team">
+            Aenean eu leo quam. Pellentesque ornare sem lacinia quam venenatis
+            vestibulum. Etiam porta sem malesuada magna mollis euismod. Fusce
+            dapibus, tellus ac cursus commodo, tortor mauris condimentum nibh,
+            ut fermentum massa justo sit amet risus. Morbi leo risus, porta ac
+            consectetur ac, vestibulum at eros. Cras justo odio, dapibus ac
+            facilisis in, egestas eget quam. Praesent commodo cursus magna, vel
+            scelerisque nisl consectetur et. Morbi leo risus, porta ac
+            consectetur ac, vestibulum at eros. Donec sed odio dui. Integer
+            posuere erat a ante venenatis dapibus posuere velit aliquet. Aenean
+            eu leo quam. Pellentesque ornare sem lacinia quam venenatis
+            vestibulum.
+          </StyledSectionHeader>
+          <TeamWrapper>
+            {/* {TEAM.map((member) => {
+              return (
+                <Image description={member.name} textAlign="bottom">
```

Comment:
Just a proactive comment, guessing this didn't work: If you map over an array, you need to set the `key` property to every element. 

```
<Image description={member.name} textAlign="bottom" key={member.name}>
```